modes/xref/evil-collection-xref.el

```diff
@@ -50,13 +52,24 @@
 
     ;; open
     (kbd "RET") 'xref-goto-xref
+    (kbd "TAB") 'xref-quit-and-goto-xref
     (kbd "S-<return>") 'xref-show-location-at-point
     "o" 'xref-show-location-at-point
     "go" 'xref-show-location-at-point)
 
   (when (>= emacs-major-version 27)
     (evil-collection-define-key 'normal 'xref--xref-buffer-mode-map
-      "gr" 'xref-revert-buffer)))
+      "gr" 'xref-revert-buffer))
+
+  (when (>= emacs-major-version 28)
+    (evil-collection-define-key 'normal 'xref--xref-buffer-mode-map
+      "[[" 'xref-prev-group
```

Comment:
I wonder if this conflicts with unimpaired style bindings.

Reply:
As far as I known, unimpaired won't/shouldn't use `[[` and `]]`. Those are used in vim motion state.